i18n/en.toml

```diff
@@ -1,195 +1,188 @@
 
-[whatsnext_heading]
-other = "What's next"
+[docs_label_users]
+other = "Users"
 
 [feedback_heading]
 other = "Feedback"
 
+[feedback_no]
+other = "No"
+
 [feedback_question]
 other = "Was this page helpful?"
 
 [feedback_yes]
 other = "Yes"
 
-[feedback_no]
-other = "No"
-
 [latest_version]
 other = "latest version."
 
-[version_check_mustbe]
-other = "Your Kubernetes server must be version "
-
-[version_check_mustbeorlater]
-other = "Your Kubernetes server must be at or later than version "
-
-[version_check_tocheck]
-other = "To check the version, enter "
-
-[caution]
-other = "Caution:"
-
-[note]
-other = "Note:"
-
-[warning]
-other = "Warning:"
+[layouts_blog_pager_prev]
+other = "<< Prev"
 
-[main_read_about]
-other = "Read about"
+[layouts_blog_pager_next]
+other = "Next >>"
 
-[main_read_more]
-other = "Read more"
+[layouts_case_studies_list_tell]
+other = "Tell your story"
 
-[main_github_invite]
-other = "Interested in hacking on the core Kubernetes code base?"
+[layouts_docs_glossary_aka]
+other = "Also known as"
 
-[main_github_view_on]
-other = "View On GitHub"
+[layouts_docs_glossary_description]
+other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to K8s, as well as more general terms that provide useful context."
 
-[main_github_create_an_issue]
-other = "Create an Issue"
+[layouts_docs_glossary_deselect_all]
+other = "Deselect all"
 
-[main_community_explore]
-other = "Explore the community"
+[layouts_docs_glossary_click_details_after]
+other = "indicators below to get a longer explanation for any particular term."
 
-[main_kubernetes_features]
-other = "Kubernetes Features"
+[layouts_docs_glossary_click_details_before]
+other = "Click on the"
 
-[main_cncf_project]
-other = """We are a <a href="https://cncf.io/">CNCF</a> graduated project</p>"""
+[layouts_docs_glossary_filter]
+other = "Filter terms according to their tags"
 
-[main_kubeweekly_baseline]
-other = "Interested in receiving the latest Kubernetes news? Sign up for KubeWeekly."
+[layouts_docs_glossary_select_all]
+other = "Select all"
 
-[main_kubernetes_past_link]
-other = "View past newsletters"
+[layouts_docs_partials_feedback_improvement]
+other = "suggest an improvement"
 
-[main_kubeweekly_signup]
-other = "Subscribe"
+[layouts_docs_partials_feedback_issue]
+other = "Open an issue in the GitHub repo if you want to "
 
-[main_contribute]
-other = "Contribute"
+[layouts_docs_partials_feedback_or]
+other = "or"
 
-[main_edit_this_page]
-other = "Edit This Page"
+[layouts_docs_partials_feedback_problem]
+other = "report a problem"
 
-[main_page_history]
-other ="Page History"
+[layouts_docs_partials_feedback_thanks]
+other = "Thanks for the feedback. If you have a specific, answerable question about how to use Kubernetes, ask it on"
 
-[main_page_last_modified_on]
-other = "Page last modified on"
+[layouts_docs_search_fetching]
+other = "Fetching results.."
```

Comment:
I understand that these text snippets are taken from a number of places on the site.
Should `layouts_docs_search_fetching` be set to `Fetching results ...`

Reply:
Same as above. It is a result of copy-paste. Fixed as you suggested.